### ISSUE TITLE
[Feat] Create semester model

### ISSUE THREAD

**Issue by Aastha-S-Rai** [tcet-opensource/erp-backend#281]
# Description
Create a file `semester.js` in models folder
Add schema with required key and value pairs

## Proposed Solution
```
number: int ENUM: [1..8]  
type: virtual ENUM: [Odd, Even]
academicYear: str 
startDate: Date
endDate: Date
```

**Comment by deepti-50** [comment]
Assign this issue to me

**Comment by TejasNair9977** [comment]
closed in #324
